package.json

```diff
@@ -80,7 +90,8 @@
     "storybook": "ace storybook",
     "build": "yarn build:prod && yarn storybook --static",
     "build:prod": "cross-env NODE_ENV=production NODE_OPTIONS='--max_old_space_size=8192' ace bundle --middleware src/main/graphql-server/server.js",
-    "prestart": "ace disable-idp"
+    "prestart": "ace disable-idp",
+    "build:library": "webpack --mode=production"
```

Comment:
I think since ace already owns the webpack config, you should trying implementing a new command there to build the source.

Reply:
Personally, I think that making a command like this re-usable isn't worth investing the time, given that we are moving towards replacing ace with nextJS. I also think that if you're making a command and effectively duplicating the config properties of webpack, you might as well use webpack directly.

Reply:
@willwill96 The point of ace is to prevent duplicating the webpack config, so the command wouldn't be copying anything, just reusing the config in a different way. Other projects that leverage ace would probably want something similar where they export their code as a library. As far as moving to next.js, it's doable but it's going to be a lot effort.